### PR TITLE
Instruction and task improvements

### PR DESCRIPTION
**Pull request by haemmer-dave**
This relates to https://github.com/exercism/csharp/issues/1951.
Clarified how to access inner type and role of "Telemetry" in instructions for task 1.

### CODE REVIEW

File: exercises/concept/remote-control-cleanup/.docs/instructions.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Some "other" developers have been working on the remote control car project [exe
 
 ## 1. Separate concerns between the car itself and the telemetry system
 
-Refactor the `RemoteControlCar` class to move to separate out telemetry methods and properties.
+Refactor the `RemoteControlCar` class to move to separate out telemetry methods and properties accessed via a `Telemetry` property.
```
Comment (ErikSchierboom) [inline exercises/concept/remote-control-cleanup/.docs/instructions.md:7]: I like this!

File: exercises/concept/remote-control-cleanup/.docs/introduction.md [hunk 2]
```diff
@@ -17,4 +17,18 @@ var inner = new Outer.CInner();
...
+    public class CInner
+    {
+        private Outer parent;
+
+        public CInner(Outer parent) {
+            this.parent = parent
+        }
+    }
+}
+```
```
Comment (ErikSchierboom) [inline exercises/concept/remote-control-cleanup/.docs/introduction.md:34]: I'm not sure about this change. It feels a bit like overexplaining things to me, whereas we prefer to have students figure things out for themselves where possible.

Comment (haemmer-dave) [reply exercises/concept/remote-control-cleanup/.docs/introduction.md:34]: Hi Erik. Thanks for your feedback. My understanding is that the Introduction section should provide some theory necessary for the exercise. Is that correct? I found the theory a bit lacking which is why I added the explanation. Would you be okay with adding a link to the C# documentation instead?

Comment (haemmer-dave) [reply exercises/concept/remote-control-cleanup/.docs/introduction.md]: @ErikSchierboom I reverted this addition

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**